components/action-mgt/org.wso2.carbon.identity.action.execution/src/main/java/org/wso2/carbon/identity/action/execution/util/APIClient.java

```diff
@@ -191,14 +235,16 @@ private void handleServerError(ActionInvocationResponse.Builder builder, HttpEnt
             if (errorResponse != null) {
                 builder.response(errorResponse);
             } else {
-                builder.errorLog("Failed to execute the action request. Received: " + statusCode);
+                builder.errorLog("Failed to execute the action request for status code " + statusCode
```

Comment:
This should say an error is recieved with a particular status code

Reply:
But here error response is NULL right ? So is it okay to say  "Failed to execute the action request. Received status code: " {statusCode}

Reply:
Updated in 87dc19a8440c8cf5f59ee103517d0fea536328fc